lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -22,6 +22,14 @@ async function runA11yChecks() {
   /** @type {import('axe-core/axe')} */
   // @ts-expect-error axe defined by axeLibSource
   const axe = window.axe;
+  const application = `lighthouse-${Math.random()}`;
+  axe.configure({
+    branding: {
+      application,
+    },
+    // @ts-ignore
+    noHtml: true,
```

Comment:
I guess this is fine since `Accessibility` isn't in our `PublicGathererArtifacts`, but we also ship back our `snippet`s which except for eliding are the same thing verbatim, so we aren't exactly creating Fort Knox over here :)